scripts/release.sh

```diff
@@ -151,24 +151,24 @@ function write_component() {
     write "  ${project}: ${commit_hash}"
 }
 
-function advance_branch() {
+function advance_from_branch() {
```

Comment:
Can we have `advance_to_X` instead?

It will be more aligned with what the functions actually do, hence clearer.